<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Return Sheet</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="tracking_id_lbl">
        <property name="text">
         <string>Tracking id:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="user_input"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="return_type_lbl">
        <property name="text">
         <string>Return Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="return_type_input"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="record_button">
        <property name="text">
         <string>Record Next</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_return_sheet_button">
        <property name="text">
         <string>Save Return Sheet</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="return_data_output">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
